src/controllers/QueryRunner.ts

```diff
@@ -295,7 +295,7 @@ export default class QueryRunner {
                         for (let row of result.resultSubset.rows) {
                             // iterate over the cells we want from that row
                             for (let cell = range.fromCell; cell <= range.toCell; cell++) {
-                                if (self.shouldRemoveNewLines() && row[cell]) {
+                                if (row[cell] && self.shouldRemoveNewLines()) {
```

Comment:
you're right
http://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in